docker/_include.sh

```diff
@@ -1,6 +1,6 @@
 DOCKER_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-if [ `uname` == 'Linux' ]; then
+if [ `uname` == 'Linux' -o `uname` == 'Darwin' ]; then
```

Comment:
I'm experiencing an issue where elasticsearch and postgres won't restart on ```./dockerhq.sh bootstrap```, and I've tracked it down to this commit.  I'm using OSX 10.11.3.  Not sure what's wrong though.

fwiw the error in my elasticsearch logs is 
```
elasticsearch_1 | [2016-03-23 19:53:43,973][INFO ][node                     ] [Stegron] version[1.7.4], pid[1], build[0d3159b/2015-12-15T16:45:04Z]
elasticsearch_1 | [2016-03-23 19:53:43,974][INFO ][node                     ] [Stegron] initializing ...
elasticsearch_1 | [2016-03-23 19:53:44,129][INFO ][plugins                  ] [Stegron] loaded [], sites [head]
elasticsearch_1 | {1.7.4}: Initialization Failed ...
elasticsearch_1 | - ElasticsearchIllegalStateException[Failed to created node environment]
elasticsearch_1 | 	AccessDeniedException[/usr/share/elasticsearch/data/Nicholass-MacBook-Pro-2.local]
```

fyi @emord @benrudolph